src/scripts/generate-imported-targets-from-snyk.ts

```diff
@@ -40,6 +40,15 @@ export function projectToTarget(
     name,
   };
 }
+export function bitbucketServerProjectToTarget(
+  project: Pick<SnykProject, 'name' | 'branch'>,
+): Target {
+  const [projectKey, repoSlug] = project.name.split(':')[0].split('/');
```

Comment:
what about the branch? You take a branch in but not return it. Do we log the branch after project creation? If we do the target we generate here needs to match or we will not correctly identify the targets as identical.

List imported command should be generating an identical format to `logImportedTargets` you can check by kicking off an import for a BBS repo and see what you get?

Reply:
Done